lua/autorun/server/sv_precision_align.lua

```diff
@@ -347,29 +347,34 @@ local function Stack_Undo( undo_table ) -- if table.Count( undo_table ) == 0 the
 	table.Empty( undo_table )
 end
 
-// Handle stack queueing (called by timer)
+-- Handle stack queueing (called by timer)
 local stackID_old
 
 local function Stack_Loop()
 	if #stack_queue > 0 and processing then
-		local r, e = pcall( Queue_Process )
+
+	    -- `result` can be an error table or the return value of Queue_Process
+	    -- Expect Queue_Process to return a table of info, or `false`
+		local success, result = pcall( Queue_Process )
+
+		local wasSuccessful = success and result ~= false

```

Comment:
Wouldn't result be a vararg not a boolean?

Reply:
Result can be `false` sometimes if the function breaks out early (for example, if creating the Entity failed, it'll return `false`)

It's confusing. `success` will always be a boolean. 
If `success` is `false`, then `result` will be a table with error information.
If `success` is `true`, then `result` will either be a boolean (if it broke out early and returned false) or a table with the expected data.

Reply:
gotcha, in that case, this looks fine to me